spec/rmagick/image/chromaticity_writer_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe Magick::Image, '#chromaticity=' do
+  it "allows setting the chromaticity" do
+    image = build_image
+
+    red_primary = Magick::Primary.new(70, 200, 0)
+    green_primary = Magick::Primary.new(200, 44, 0)
+    blue_primary = Magick::Primary.new(60, 22, 0)
+    white_point = Magick::Primary.new(77, 43, 0)
+    chromaticity = Magick::Chromaticity.new(red_primary, green_primary, blue_primary, white_point)
+    image.chromaticity = chromaticity
+
+    expect(image.chromaticity.red_primary).to eq(red_primary)
+    expect(image.chromaticity.green_primary).to eq(green_primary)
+    expect(image.chromaticity.blue_primary).to eq(blue_primary)
+    expect(image.chromaticity.white_point).to eq(white_point)
+  end
+
+  it "sets the z values to 0" do
```

Comment:
Why is this true? Why does it ignore the given `z` value and set it to 0? You can see it doing this in `Export_ChromaticityInfo` inside of `rmstruct.c`.